swig/cpp/src/Tree_Data.hpp

```diff
@@ -268,6 +268,8 @@ class Data_Node_Anydata : public Data_Node
     ~Data_Node_Anydata();
     /** get value_type variable from [lyd_node_anydata](@ref lyd_node_anydata)*/
     LYD_ANYDATA_VALUETYPE value_type() {return ((struct lyd_node_anydata *) node)->value_type;};
+    /** get value variable from [lyd_node_anydata](@ref lyd_node_anydata)*/
+    LYD_ANYDATA_VALUE value() {return ((struct lyd_node_anydata *) node)->value;};
     //union value
```

Comment:
this is probably kind of TODO comment you are fixing, so it should be removed

Reply:
done